amazon-kinesis-client/src/test/java/software/amazon/kinesis/config/BasicReleaseCanaryConfig.java

```diff
@@ -0,0 +1,108 @@
+package software.amazon.kinesis.config;
+
+import lombok.extern.slf4j.Slf4j;
+import software.amazon.awssdk.http.Protocol;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.kinesis.common.InitialPositionInStream;
+import software.amazon.kinesis.common.InitialPositionInStreamExtended;
+import software.amazon.kinesis.retrieval.RetrievalConfig;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+/**
+ * Basic config for a release canary (streaming) with default settings
```

Comment:
Since this file was combined with KCLAppConfig, these comments are now addressed in that file. I have updated the description at the top. Streaming (vs polling) refers to the protocol for getting data. In the streaming scenario, the connection between the producer and the consumer is made once and the producer sends data continuously while the connection is maintained. In the polling scenario, the consumer calls the producer each time it wants records to process.